drgn/helpers/linux/tc.py

```diff
@@ -10,11 +10,102 @@
 """
 
 import operator
+from typing import Iterator
 
 from drgn import NULL, IntegerLike, Object
 from drgn.helpers.linux.list import hlist_for_each_entry, list_for_each_entry
 
-__all__ = ("qdisc_lookup",)
+__all__ = (
+    "for_each_tcf_chain",
+    "for_each_tcf_proto",
+    "get_tcf_chain_by_index",
+    "get_tcf_proto_by_prio",
+    "qdisc_lookup",
+)
+
+
+def for_each_tcf_chain(block: Object) -> Iterator[Object]:
+    """
+    Iterate over all TC filter chains on a block.
+
+    This is only supported since Linux v4.13.
```

Comment:
I'm not familiar with TC at all, but looking at the commits you reference here, it looks like TC filter chains didn't exist at all before 4.13, is that correct? Maybe we can say something like "This is only supported since Linux v4.13, before which TC filter chains didn't exist.".